logupload.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+#
+# Copyright 2019 Joyent, Inc.
+#
+
+#
+# Push /var/log/manta/upload/... log files up to Manta.
```

Comment:
I think it would be clearer to name this "logrotateandupload.sh" (or similar). I was initially surprised that it was handling the `logadm` call itself.

This top comment should be updated.